calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.1-release-notes.mdx

```diff
@@ -1,4 +1,4 @@
-27 Jan 2023
+xx Mar 2023
```

Comment:
I think this change belongs in the other file, `_v3.15.2-release-notes.mdx`.

Reply:
Fixed